src/build-2024/02 - Add Chat History/README.md

```diff
@@ -0,0 +1,10 @@
+﻿## Console Chat Application with Stateful Chat AI
+
+This console application demonstrates a basic chat interface using OpenAI's GPT model (such as GPT-3.5-Turbo or GPT-4-Turbo) through the Microsoft Semantic Kernel. The application maintains a stateful chat interface, where it continuously takes user input and provides responses generated by the AI model, while remembering previous contexts.
+### Chat Service
+`kernel.GetRequiredService<IChatCompletionService>()` retrieves the chat completion service from the kernel.
+
+#### Chat History
+Named `chatHistory` is used to store the conversation history. Each user message and AI response is appended to this list, maintaining the context of the conversation.
+
+By maintaining a chat history, the application ensures that the AI model can remember previous contexts and provide more coherent and contextually relevant responses.
```

Comment:
I assume this is a still a work in progress :)

Reply:
yes it is